### ISSUE TITLE
HTTP 500 Error from Zokrates when try to complete withdraw

### ISSUE THREAD

**Issue by luanpontolio** [EYBlockchain/starlight#245]
Hey everyone,

We found a possible problem when trying to perform a completed withdrawal of all the funds from `EscrowShield`

Zokrates
```
zokrates  | info: Compute witness...
zokrates  | info: Generate proof...
zokrates  | warn: Unable to locate file:   '/app/output/withdraw/withdraw_5109cfabefc3f17702cc1dabbfbc8739cd4afe5def569797dbd946bb15c0c65b_proof.
```

Zapp
```
zapp    | Calling zokrates/generate-proof(withdraw)
zapp    | error: Error: Request failed with status code 500 Error: Error: Request failed with status code 500
zapp    |     at generateProof (file:///app/orchestration/common/zokrates.mjs:40:9)
zapp    |     at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
zapp    |     at async WithdrawManager.withdraw (file:///app/orchestration/withdraw.mjs:249:17)
zapp    |     at async ServiceManager.service_withdraw (file:///app/orchestration/api_services.mjs:108:39)
```

Thanks for the help!

**Comment by luanpontolio** [comment]
Yep, I checked here and have the pair of commitments(`isNullified=false`) and the amount was equal to my balance.
The scenario was that I made two deposits with the amount of `8000` each, bringing my current balance to `16000`. When I try to withdraw the entire amount, the proof generator endpoint from Zokrates returns the 500 error. However, if I withdraw only 15999 and leave $1 behind, the withdrawal goes through successfully.  As you mention maybe this `assert` could be the cause.
